cmd/dagger/flags.go

```diff
@@ -290,7 +290,14 @@ func (v *secretValue) Get(ctx context.Context, c *dagger.Client) (any, error) {
 	case envSecretSource:
 		envPlaintext, ok := os.LookupEnv(v.sourceVal)
 		if !ok {
-			return nil, fmt.Errorf("secret env var not found %q", v.sourceVal)
+			// Don't show the entire env var name, in case the user accidentally passed the value instead...
+			// This is important because users originally *did* have to pass the value, before we changed to
+			// passing by name instead.
+			key := v.sourceVal
+			if len(key) >= 4 {
+				key = key[:3]
+			}
+			return nil, fmt.Errorf("secret env var not found %q...", key)
```

Comment:
The linting failure is incredibly dumb and just doesn't like `...` at the end of an error message. Can put a `//nolint:ST1005` above this line to ignore it or I suppose change to `fmt.Errorf("secret env var not found %q", key+"...")` if we want to appease it.

Otherwise LGTM!